pkgs/by-name/in/invoiceplane/package.nix

```
@@ -0,0 +1,79 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  nixosTests,
+  fetchYarnDeps,
+  nodejs,
+  php,
+  yarnConfigHook,
+  yarnBuildHook,
+  yarnInstallHook,
+}:
+
+php.buildComposerProject (finalAttrs: {
+  pname = "invoiceplane";
+  version = "1.6.1";
+
+  src = fetchFromGitHub {
+    owner = "InvoicePlane";
+    repo = "InvoicePlane";
+    rev = "refs/tags/v${finalAttrs.version}";
+    hash = "sha256-rVt3HkAh3zBVwt1a4ZGiuC/Khvb3Ugk42hT5mul4qRA=";
+  };
+
+  vendorHash = "sha256-qYjNn7VQGkUfJ74coUpUOfAOkgi3eLoo/ITpcOcTenk=";
+
+  nativeBuildInputs = [
+    yarnConfigHook
+    yarnBuildHook
+    yarnInstallHook
+    # Needed for executing package.json scripts
+    nodejs
+  ];
+
+  offlineCache = fetchYarnDeps {
+    yarnLock = ./yarn.lock;
+    hash = "sha256-t9GYFqKY3YZ/l5LZXIHIgehgvxIbMMuYLwXvvdQaryM=";
+  };
+
+  # Upstream yarn.lock and package.json out of sync
+  # https://github.com/NixOS/nixpkgs/pull/353783#discussion_r1830882396
```

Feel free to include the new issue in the upstream repo in the comment here.
```suggestion
  # https://github.com/InvoicePlane/InvoicePlane/issues/1109
```